src/GafferOSL/OSLShader.cpp

```diff
@@ -84,9 +104,19 @@ struct ShadingEngineCacheGetterKey
 	{
 	}
 
-	ShadingEngineCacheGetterKey( const OSLShader *s )
-		:	shader( s ), hash( s->attributesHash() )
+	ShadingEngineCacheGetterKey( const OSLShader *s, const CompoundObject *targetA )
```

Comment:
The `targetA` name seems to be a hangover from previous naming - should just be `substitutions`.